12/umbraco-commerce/reference/storefront-api/endpoints/checkout.md

```diff
@@ -0,0 +1,27 @@
+# Checkout Endpoints
+
+The checkout endpoints provide ways of performing a checkout process against an Order. The Storefront API supports two ways of checking out an order, one using hosted checkout pages, and a more advanced option for inline payment processing.
+
+## Hosted
+
+With the hosted checkout flow it is required that before redirecting to the payment gateway a checkout token should be generated. This token is passed to the pay endpoint to ensure that only the given order can be processed in response to the checkout request. The pay endpoint should then be launched in a WebView with this token which will then redirect to the given Orders payment gateway for payment capture. To determine the outcome of the payment developers should monitor the WebViews URL which will be redirected to the same pay endpoint URL with either a `/completed`, `/canceled` or `/errored` suffix.

```

Comment:
Hi @mattbrailsford 👋 

We have a styleguide to follow to make the documentation more readable, where one of guides is to write [shorter sentences ](https://docs.umbraco.com/welcome/contribute/style-guide#long-sentences )with max 25 words. 

It seems in this case, these 2 sentences have more than 25 words. Can you please look into them? 😊 

_"The pay endpoint should then be launched in a WebView with this token which will then redirect to the given Orders payment gateway for payment capture." -_- my suggestion here would be to remove first "then"

To determine the outcome of the payment developers should monitor the WebViews URL which will be redirected to the same pay endpoint URL with either a `/completed`, `/canceled` or `/errored` suffix. - my suggestion here would be to split this into 2 sentences like so:
 "to determine the outcome of the payment developers should monitor the WebViews URL. Then they will be will be redirected to the same pay endpoint URL with either a `/completed`, `/canceled` or `/errored` suffix."

Reply:
Updated 👍